vendor/bat-native-ads/src/bat/ads/internal/legacy_migration/client/legacy_client_migration_unittest.cc

```diff
@@ -19,7 +19,7 @@ namespace client {
 namespace {
 
 constexpr uint64_t kClientJsonHash = 1891112954;
-constexpr uint64_t kMigratedClientJsonHash = 1943308255;
+constexpr uint64_t kMigratedClientJsonHash = 1204433941;
```

Comment:
Changed because we removed `version_code` from the `client.json` and we need to regenerate the hash